docker/docker-compose.test.yml

```diff
@@ -23,14 +34,6 @@ services:
       dockerfile: ./docker/Dockerfile.test
     links:
       - db_test
+      - musicbrainz_db
```

Comment:
The `musicbrainz_db` container should finish setup for tests before the tests container starts. Let's improve this, you can try with sleeping 10-20 seconds before starting the tests. (Hint: Look at the `CMD` command in `docker/Dockerfile.test`)